openmc/mgxs/mgxs.py

```diff
@@ -919,23 +919,6 @@ def load_from_statepoint(self, statepoint):
                   'linked with a summary file'
             raise ValueError(msg)
 
-        # Override the domain object that loaded from an OpenMC summary file
-        # NOTE: This is necessary for micro cross-sections which require
-        # the isotopic number densities as computed by OpenMC
```

Comment:
I thought this was fine since the "test_mgxs_library_nuclides" test still passes, but it appears that is simply the case  because the macro rather than the micro MGXS are stored in the test results. I need to add this code block back in, but will need to think about how to most efficiently do so since it is the cause of the huge performance degradation in #833.